cameraview/src/main/java/com/otaliastudios/cameraview/CameraListener.java

```diff
@@ -73,9 +73,11 @@ public void onPictureTaken(byte[] jpeg) {
      * </code>
      *
      * @param video file hosting the mp4 video
+     * @param hasReachedMaxFileSize returns true if the user has set a max file size for videos and
+     *                                if the video ended because of the file size limit.
      */
     @UiThread
-    public void onVideoTaken(File video) {
+    public void onVideoTaken(File video, boolean hasReachedMaxFileSize) {
```

Comment:
As said, I think this is unacceptable at the moment. We can't force everyone to rewrite their code. But in the future I will change the signature with a `VideoResult` so we can add everything.